xbmc/pvr/timers/PVRTimerInfoTag.cpp

```diff
@@ -764,21 +757,6 @@ CDateTime CPVRTimerInfoTag::EndAsLocalTime(void) const
   return retVal;
 }
 
-bool CPVRTimerInfoTag::IsEndAtAnyTime(void) const
-{
-  time_t time = 0;
-  CDateTime stop(m_StopTime);
-  stop.GetAsTime(time);
-  return time == 0;
-}
-
-void CPVRTimerInfoTag::SetEndAtAnyTime(void)
-{
-  time_t time = 0;
-  CDateTime stop(time);
-  SetEndFromUTC(stop);
-}
-
```

Comment:
OK - but I don't understand what 'Serialize' is supposed to do, so I'd definately get it wrong. What's missing / incorrect?